spec/rails_helper.rb

```diff
@@ -118,7 +118,7 @@
   end
 
   if ENV['CI']
-    junit = "tmp/junit/spec-#{[ENV['MULTIJOB_KIND'], Process.pid].compact.join('-')}/spec.xml"
+    junit = "tmp/junit/spec, Process.pid].compact.join('-')}/spec.xml"
```

Comment:
I think they are not used but as circleci use multiple containers to run tests I wonder if those are not overriden

See if we can replace it by one of the built-in env variables

https://circleci.com/docs/2.0/env-vars/#built-in-environment-variables